src/settings.ts

```diff
@@ -151,7 +162,44 @@ export class ConnectionSettingsService {
       .update(SONARCLOUD_CONNECTIONS_CATEGORY, scConnections, VSCode.ConfigurationTarget.Global);
   }
 
-  async addTokensFromSettingsToSecureStorage(sqConnections: SonarQubeConnection[]) {
+  async addSonarCloudConnection(connection: SonarCloudConnection) {
+    const connections = this.getSonarCloudConnections();
+    const newConnection: SonarCloudConnection = { organizationKey: connection.organizationKey };
+    if (connection.connectionId !== undefined) {
+      newConnection.connectionId = connection.connectionId;
+    }
+    if (connection.disableNotifications) {
+      newConnection.disableNotifications = true;
+    }
```

Comment:
equivalent to `newConnection.disableNotifications = connection.disableNotifications` ?

Reply:
Not exactly: we only want to add the `disableNotifications` attribute when it is `true` (`false` is the default value).